src/addRemove.test.js

```diff
@@ -0,0 +1,23 @@
+import { addTodo, removeTodo } from './modules/functions.js';
+
+// set up the initial container with zero tasks
+document.body.innerHTML = '<div class="main"><header>Todays To Do</header><hr><div class="addInput-container"><input type="text" class="addInput" placeholder="Press enter key to Add to your list..."><div class="addBtn">&#8617;</div></div><hr><div class="todoContainer"><!-- here all the content will be created dinamically  --></div><footer class="clrBtn">Clear all completed</footer></div>';
+const container = document.querySelector('.todoContainer');
+
+describe('add function', () => {
+  test('should add one task to the container', () => {
+    addTodo('Test task 1');
+    expect(container.children.length).toBe(1);
+  });
+
+  test('the container should contain a task called Test task 2', () => {
+    addTodo('Test task 2');
+    expect(container.innerHTML).toContain('Test task 2');
+  });
+
+  test('should add a third task to the container', () => {
+    addTodo('Test task 3');
+    const container = document.querySelector('.todoContainer');
+    expect(container.children.length).toBe(3);
+  });
+});
```

Comment:
- As the previous reviewer suggested, you still have not met the project requirements. You are supposed to add a test file for editing a task, editing a task completed status and the clear all completed button. None of these tests are present in your project. Please add them🙏
- Also, one of your test suites is failing as seen in the screenshot below. Please make sure that all tests are passing
<img width="435" alt="image" src="https://user-images.githubusercontent.com/37116322/232208833-d5921904-d8a6-43fa-ac84-15d341528749.png">